README.md

```
@@ -37,24 +39,49 @@ The reward mechanism for Subnet 2 scores the initial AI predictions based on the
 ## Quickstart
 
-Run the below command to install Subnet 2 and it's dependencies.
+### Prerequisites
+
+| Tool | Description |
+|------|-------------|
+| [Rust](https://rustup.rs/) | Required to build from source. Install via `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \| sh` |
+| [PM2](https://pm2.keymetrics.io/) | Process manager used to run and monitor the binaries in the background |
+| [btcli](https://docs.bittensor.com/getting-started/installation) | CLI for interacting with the Bittensor network (wallet creation, registration) |
+
+Alternatively, use Docker — no Rust toolchain needed. See the [Docker instructions](#docker) below.
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)